api/checks/suites.go

```diff
@@ -121,3 +129,15 @@ func completeCheckRun(ctx context.Context, sha string, product shared.ProductSpe
 	}
 	return true, nil
 }
+
+func scheduleResultsProcessing(ctx context.Context, sha string, product shared.ProductSpec) error {
+	host := shared.GetHostname(ctx)
+	target, _ := url.Parse(fmt.Sprintf("https://%s/api/checks/%s", host, sha))
+	q := target.Query()
+	q.Set("product", url.QueryEscape(product.String()))
```

Comment:
IIRC `Values.Encode` will take care of encoding the values, so this is a double-encoding.

Please add a test case to verify.

Reply:
I was using this quite wrong anyway; cleanup makes it redundant.